src/include/common/copier_config/csv_reader_config.h

```diff
@@ -15,14 +15,18 @@ struct CSVOption {
     bool hasHeader;
     uint64_t skipNum;
     uint64_t sampleSize;
+    char listDelimiter;
```

Comment:
As we discussed, we no longer support customize list delimter, we should remove this field and use static constexpr for listDelimiter.